ctp2_code/gfx/spritesys/director.cpp

```diff
@@ -2037,43 +2004,23 @@ class DQActionBeginScheduler : public DQActionExternal
 	sint32 player;
 };
 
-DQItem::DQItem(DQAction *action)
-:
-	m_owner             (PLAYER_UNASSIGNED),
-	m_round             (0),
-	m_action            (action)
-{
-	if (g_turn)
-	{
-		m_round = static_cast<uint16>(g_turn->GetRound());
-	}
-}
-
-DQItem::~DQItem()
-{
-	//DPRINTF(k_DBG_GAMESTATE, ("Deleting item @ %lx, type=%d\n", this, m_type));
-	delete m_action;
-}
-
-#define k_MAX_DISPATCHED_QUEUE_ITEMS		1000
-
 DirectorImpl *DirectorImpl::m_instance = NULL;
 DirectorImpl::DirectorImpl(void)
 :
-	m_currentItem               (NULL),
-	m_activeEffectList          (new tech_WLList<EffectActor *>),
-	m_tradeActorList            (new tech_WLList<TradeActor *>),
-	m_masterCurTime             (0),
-	m_lastTickCount             (0),
-	m_timeLogIndex              (0),
-	m_averageElapsed            (0),
-	m_averageFPS                (k_DEFAULT_FPS),
-	m_paused                    (FALSE),
-	m_processingActiveEffects   (FALSE),
-	m_itemQueue                 (new PointerList<DQItem>),
-	m_itemWalker                (new PointerList<DQItem>::Walker),
-	m_pendingGameActions        (0),
-	m_endTurnRequested          (false)
+	m_currentAction				(NULL),
+	m_activeEffectList			(new tech_WLList<EffectActor *>),
+	m_tradeActorList			(new tech_WLList<TradeActor *>),
+	m_masterCurTime				(0),
+	m_lastTickCount				(0),
+	m_timeLogIndex				(0),
+	m_averageElapsed			(0),
+	m_averageFPS				(k_DEFAULT_FPS),
+	m_paused					(FALSE),
+	m_processingActiveEffects	(FALSE),
+	m_actionQueue				(new PointerList<DQAction>),
+	m_actionWalker				(new PointerList<DQAction>::Walker),
+	m_pendingGameActions		(0),
+	m_endTurnRequested			(false)
```

Comment:
Now this is harder to read. Please keep these things aligned. Keep in mind that tabs lengths is a setting in your editor and does not need to be at four spaces. In general uses tabs for things that are not screwed up by any tab setting. That are the things on the left. And use spaces for things that are affected by the tab settings like those on the right.

Reply:
> In general uses tabs for things that are not screwed up by any tab setting.  That are the things on the left. And use spaces for things that are affected by the tab settings like those on the right.

Thanks I will follow this guideline and update Director classes.

May I propose to not using `tabs` as most editors now support inject `spaces` for `tab`. I do understand that would be another change in style so a huge change. We may plan to update all files once (UTF-8, no-tabs, maybe other changes).